Added bugset link to main OpenHatch page.

mgc1998 opened the pull request.

SanketDG commented on mysite/base/templates/base/base.html:
```diff
@@ -190,6 +190,7 @@
 		<ul>
                   <li><a href="/wiki/">wiki</a></li>
                   <li><a href="http://forum.openhatch.org/">forum</a></li>
+				  <li><a href="https://openhatch.org/bugsets/">create a bugset</a></li>
```
What's the strange extra indent for?

ehashman commented:
Hi @mgc1998! I've taken a look at your commits here. Thanks so much for your contribution to OpenHatch!

Two things:

- Can you please use the relative link to bugsets instead of the fully qualified domain name? i.e. `href='/bugsets/'`
- Can you please squash your changes into a single commit using `git rebase`? Check out [this documentation we have](http://openhatch.readthedocs.org/en/latest/advanced/working_with_git.html#working-with-git) and feel free to ask me questions either here or on IRC if you need help with that.

mgc1998 commented:
Hello Ms. Hashman

I am (clearly) new to openhatch so thanks for your help. I changed the link
to bugsets to href='/bugsets/', and tried my best to rebase the changes
into one commit, but I think I did it wrong. The first time I had forgotten
to save my edit so it still had the full domain name in the code, but I
think all of the spacing errors were fixed in one commit. I then tried to
go back and save my edit then rebase all three into one commit, but this is
where I ran into a couple of problems. When I type in "git rebase
--interactive HEAD~7" I got this screen: [image: Inline image 1]. First of
all, only the last three lines are commits that I made, so I am not sure
where the rest came from. Secondly, I already squashed the "fixed spacing"
commit into the one before it ("Added bugset link to main OpenHatch page"),
so I don't know if they are supposed to show up together or separately.
Then when I change the "pick" in front of "final addition" (the edit that
should fix the domain name) to "s" to squash it in with the other two, I
run into a bunch of errors: [image: Inline image 2][image: Inline image 3]and
I am not sure where to go from here.

Thank you so much for your help as I understand that I am probably making
some silly error, so I really appreciate it. Let me know what you think
when you get a chance.

Regards,
Callie Cruff

On Sat, Jan 2, 2016 at 1:07 AM, Elana Hashman <notifications@github.com>
wrote:

> Hi @mgc1998 <https://github.com/mgc1998>! I've taken a look at your
> commits here. Thanks so much for your contribution to OpenHatch!
>
> Two things:
>
>    - Can you please use the relative link to bugsets instead of the fully
>    qualified domain name? i.e. href='/bugsets/'
>    - Can you please squash your changes into a single commit using git
>    rebase? Check out this documentation we have
>    <http://openhatch.readthedocs.org/en/latest/advanced/working_with_git.html#working-with-git>
>    and feel free to ask me questions either here or on IRC if you need help
>    with that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openhatch/oh-mainline/pull/1777#issuecomment-168367353>
> .
>
